linux_os/guide/system/software/updating/ensure_gpgcheck_local_packages/bash/rhel7.sh

```diff
@@ -1,8 +1,4 @@
 # platform = Red Hat Enterprise Linux 7
+. /usr/share/scap-security-guide/remediation_functions
 
-if grep --silent ^localpkg_gpgcheck /etc/yum.conf ; then
-        sed -i "s/^localpkg_gpgcheck.*/localpkg_gpgcheck=1/g" /etc/yum.conf
-else
-        echo -e "\n# Set localpkg_gpgcheck to 1 per security requirements" >> /etc/yum.conf
-        echo "localpkg_gpgcheck=1" >> /etc/yum.conf
-fi
+replace_or_append '/etc/yum.conf' '^localpkg_gpgcheck' '1' '@CCENUM@'
```

Comment:
With implied property `pkg_manager_config_file` introduced in #3251 remediation for `fedora` and `rhel7` can be merged with jinja.

Reply:
@yuumasato Yes, I agree. After #3251 is merged I can rework this.